ampersand-chart.js

```diff
@@ -456,6 +467,10 @@
           }
         }]
       });
+      filterTrackerState.onApply = function(props) { 
+        this.model.filterOnApply.bind(this.model)(props);
```

Comment:
You should be able to just say `this.model.filterOnApply(props);`.  That automatically binds to `this.model`.